spec/features/result_filters/location_and_provider_spec.rb

```diff
@@ -31,39 +32,34 @@
       )
 
       stub_courses(
-        query: base_parameters.merge('filter[provider.provider_name]' => 'ACME SCITT 0'),
+        query: base_parameters.merge('filter[provider.provider_name]' => 'ACME SCITT 0', 'filter[subjects]' => '00'),
         course_count: 4,
       )
     end
 
     context 'valid provider search' do
       before do
-        results_page.load
-        results_page.area_and_provider_filter.link.click
+        filter_page.load
         filter_page.by_provider.click
         filter_page.provider_search.fill_in(with: 'ACME')
         filter_page.find_courses.click
         provider_page.provider_suggestions[0].hyperlink.click
+        subjects_page.subject_areas.first.subjects[0].checkbox.click
+        subjects_page.continue.click
       end
 
       it 'displays the courses' do
         expect(results_page.courses.first).to have_main_address
-        expect(results_page.heading.text).to eq('Teacher training courses 4 courses found')
-        expect(results_page.area_and_provider_filter.name.text).to eq('ACME SCITT 0')
-        expect(results_page.area_and_provider_filter.link.text).to eq('Change provider or choose a location')
+        expect(results_page.heading.text).to eq('4 courses found')
       end
 
       it 'retains the query parameters' do
         expect_page_to_be_displayed_with_query(
           page: results_page,
           expected_query_params: {
-            'fulltime' => 'false',
-            'parttime' => 'false',
-            'hasvacancies' => 'true',
             'l' => '3',
-            'qualifications' => %w[QtsOnly PgdePgceWithQts Other],
-            'senCourses' => 'false',
```

Comment:
Interesting one here @duncanjbrown... These default params are not in the URL when I manually walk through this and land on the results page for the first time, yet the test is able to pull them from the `current_url`? On my branch they don't appear, hence why I've removed them but behaviour, re params, locally between my branch & master is the same. Any ideas? cc @davidgisbey